pkg/apis/metrics/v1alpha2/traffic_split.go

```diff
@@ -0,0 +1,9 @@
+package v1alpha2
+
+// TrafficSplit describes which branch of a traffic split these metrics
+// represent.
+type TrafficSplit struct {
+	apex   string `json:"apex"`
+	leaf   string `json:"leaf"`
+	weight int    `json:"weight"`
```

Comment:
Reusing `TrafficSplit` name is very confusing to me, shouldn't this be reference to a existing TrafficSplit? something like:
```
type TrafficSplit struct {
*v1.ObjectReference
Backend string
}
```

What about HTTP match conditions?

Reply:
You're right, this is a confusing name.  This object is meant to indicate which branch of the traffic of traffic split these metrics represent.  Do you have any suggestions for a better name?  What about something like, `traffic_split_branch`, or `backend`?

Reply:
I think `backend` would be better.